src/librustc_mir/borrow_check/mod.rs

```diff
@@ -830,6 +835,37 @@ impl InitializationRequiringAction {
     }
 }
 
+/// A simple linked-list threaded up the stack of recursive calls in `visit_terminator_drop`.
+#[derive(Debug)]
+struct SeenTy<'a, 'gcx: 'a>(ty::Ty<'gcx>, Option<&'a SeenTy<'a, 'gcx>>);
```

Comment:
nit: `Ty` not `ty::Ty`.

Reply:
yeah I guess we've hit the number of occurrences where we can add `Ty` to the `use` at the top.